docs/faq.rst

```diff
@@ -43,6 +43,13 @@ If you want to have 2 independent backups, it is better to do it like this:
 - client machine ---borg create---> repo1
 - client machine ---borg create---> repo2
 
+.. note::
+
+    If you copy a repository with the lock held, the lock will be present in
+    the copy. Thus, before using borg on the copy from a different host,
+    you need to use :ref:`with-lock` on the copied repository, because
+    Borg is cautious and does not automatically remove stale locks made by a different host.
+
```

Comment:
http://borgbackup.readthedocs.io/en/stable/usage/lock.html

there is already a similar not there, i do not think we should duplicate it (but rather improve it at one place and link to that place where needed).